### PR TITLE
No OSX py 36 build #2

### PR DESCRIPTION
**Pull request by stuertz**

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -25,7 +24,7 @@ requirements:
     - python
     - parse >=1.6
     - six
-    - enum34
+    - enum34  # [py2k]
```
Comment (ocefpaf) [inline recipe/meta.yaml:27]: Did you rerender?

Comment (stuertz) [reply recipe/meta.yaml:27]: No, what exactly is todo? I just did a python36 build on windows, and it worked. 
Where is described what needs to be done? Does it work with windows?

I experimented with "conda smithy rerender" but that failed, deleting .git files

Comment (ocefpaf) [reply recipe/meta.yaml:27]: Take a look at our docs (and please make some comments :wink:):

https://github.com/conda-forge/conda-forge.github.io/pull/350/files

Re-rendering there will update the feedstock to include Python 3.6 in the build matrix.

Comment (stuertz) [reply recipe/meta.yaml:27]: I do get:
```
Traceback (most recent call last):
  File "C:\Users\js\AppData\Local\Continuum\Miniconda2\Scripts\conda-smithy-script.py", line 5, in <module>
    sys.exit(conda_smithy.cli.main())
  File "C:\Users\js\AppData\Local\Continuum\Miniconda2\lib\site-packages\conda_smithy\cli.py", line 271, in main
    args.subcommand_func(args)
  File "C:\Users\js\AppData\Local\Continuum\Miniconda2\lib\site-packages\conda_smithy\cli.py", line 182, in __call__
    configure_feedstock.main(args.feedstock_directory)
  File "C:\Users\js\AppData\Local\Continuum\Miniconda2\lib\site-packages\conda_smithy\configure_feedstock.py", line 703, in main
    copy_feedstock_content(forge_dir)
  File "C:\Users\js\AppData\Local\Continuum\Miniconda2\lib\site-packages\conda_smithy\configure_feedstock.py", line 589, in copy_feedstock_content
    'README'
  File "C:\Users\js\AppData\Local\Continuum\Miniconda2\lib\site-packages\conda_smithy\configure_feedstock.py", line 580, in copytree
    copy_file(s, d)
  File "C:\Users\js\AppData\Local\Continuum\Miniconda2\lib\site-packages\conda_smithy\feedstock_io.py", line 87, in copy_file
    repo.index.add([dst])
  File "C:\Users\js\AppData\Local\Continuum\Miniconda2\lib\site-packages\git\index\base.py", line 750, in add
    entries_added.extend(self._entries_for_paths(paths, path_rewriter, fprogress, entries))
  File "C:\Users\js\AppData\Local\Continuum\Miniconda2\lib\site-packages\git\util.py", line 75, in wrapper
    return func(self, *args, **kwargs)
  File "C:\Users\js\AppData\Local\Continuum\Miniconda2\lib\site-packages\git\index\util.py", line 91, in set_git_working_dir
    return func(self, *args, **kwargs)
  File "C:\Users\js\AppData\Local\Continuum\Miniconda2\lib\site-packages\git\index\base.py", line 636, in _entries_for_paths
    entries_added.append(self._store_path(filepath, fprogress))
  File "C:\Users\js\AppData\Local\Continuum\Miniconda2\lib\site-packages\git\index\base.py", line 605, in _store_path
    istream = self.repo.odb.store(IStream(Blob.type, st.st_size, stream))
  File "C:\Users\js\AppData\Local\Continuum\Miniconda2\lib\site-packages\gitdb\db\loose.py", line 233, in store
    remove(obj_path)
WindowsError: [Error 5] Zugriff verweigert: u'C:\\work\\conda\\conda-feedstocks\\parse_type-feedstock\\.git\\objects\\28\\8029dca17ee2d73d8415761cbc1da2c449ef94'

```
Which looks at first sight as an issue with gitdb and windows file system.

I'll report this to gitdb and try it again on linux during the next days :-(

see: https://github.com/gitpython-developers/gitdb/issues/37

**Pull request merged**